### DISCUSSION TITLE
How To Run Only Watchlist Sync

### DISCUSSION THREAD

**Discussion by Migz93** [Taxel/PlexTraktSync#1019] [Q&A] [answered]
Hi All, i hope you can help me with where I'm most likely being stupid.
I want to run just the watchlist sync portion from plex to trakt every hour.
I have setup a second instance of plextraktsync and that config is like this:
https://pastebin.com/bNJQ38sR

I assume this means it only runs the plex_to_trakt watchlist section but when I run it I can see it going through all my movies and all my tv shows.

It takes as long as my normal daily sync which has this config:
https://pastebin.com/ar6UDayL

Is it possible to have literally just the watchlist sync portion run by itself? Aka it pulls my watchlist down, then adds those items to trakt without having to go through my entire library?
I've seen that process within the full run take no more then 30 seconds, whereas the full run itself usually takes 10-15 minutes.
Log of just the watchlist run:
https://pastebin.com/zZtCSAx4

**Comment by simonc56** [comment, marked as answer]
> Is it possible to have literally just the watchlist sync portion run by itself?

 It's possible from version `0.21.4` ( added with PR #1040 )


~~For now it's not possible directly.~~

~~A workaround is to exclude all your Plex libraries in config file under `excluded-libraries`.~~
~~As another workaround you can choose to sync only one movie with the `--id` parameter :~~

~~`plextraktsync sync --id 4356`~~

~~(the id can be found in [the xml of a Plex item](https://github.com/Taxel/PlexTraktSync/blob/HEAD/CONTRIBUTING.md#plex-media-xml). It's named as _ratingKey_)~~

**Comment by Migz93** [comment]
Thanks for that, it turns out by running the sync --id command that it also does a full watchlist sync.
Aka if I run `plextraktsync sync --id 245517` it first syncs for the movie Samaritan and then goes on to do my full watchlist too, I've just seen it add top gun to my watchlist for example.

Although whilst testing I've just noticed it fails to remove things from my watchlist but I'll raise that separately as an issue.
Thanks again, that sync --id command will work perfectly for me.
